src/System.Security.Permissions/ref/DummyTypes.cs

```diff
@@ -0,0 +1,6 @@
+namespace System
```

Comment:
These are types we expect to live in a lower contract but aren't there yet?  If so, I think it'd be worth adding a TODO comment, ideally that includes the relevant GitHub issue number.